### ISSUE TITLE
Language support

### ISSUE THREAD

**Issue by FireCubeStudios** [FireCubeStudios/Appraisit#2]
Add support for international languages

**Comment by ktos** [comment]
I think I can take it.

I browsed through the source code for a while, and probably the first step would be to move every localizeable string into string resources. The next would be to use x:Uid for controls.

**Comment by FireCubeStudios** [comment]
ok I haven't done this before. is this the documentation to follow: https://docs.microsoft.com/en-us/windows/uwp/app-resources/localize-strings-ui-manifest ?

**Comment by ktos** [comment]
Yes, it's the basic howto on localizing - you are using x:Uid in XAML and resources using this Uid and using resources instead of hard-coded strings.

It seems you already have some helpers useful in localization: https://github.com/FireCubeStudios/Appraisit/blob/master/Appraisit/Helpers/ResourceExtensions.cs

There is also a bit documentation about different issues you may encounter: https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/globalizing-portal

See also my description of PR#19 and the whole diff to get the concept what was changed.
